lib/honeycomb/propagation.rb

```diff
@@ -47,17 +47,35 @@ def parse_v1(payload)
 
       [trace_id, parent_span_id, trace_fields, dataset]
     end
+
+    private
+
+    def parse_span_id(string)
+      INVALID_SPAN_ID = ("0" * 16).b
```

Comment:
maybe? This is a little lazy, but also checks directly against the invalid string in the w3c spec https://www.w3.org/TR/trace-context/#trace-id. We could also move the invalid ids out of `private` and unpack them, which is what they're doing in otel

Reply:
I'm not sure what you mean by move them out of private? We should pull the constants out of the method at least, but leaving them private seems good